projects/lf-documentation/src/app/app.component.css

```diff
@@ -2,10 +2,14 @@ button {
     margin-right: 5px;
 }
 .arrow-left {
-  background-color: white;
-  border-width: 1px;
-  border-style: solid;
-  border-color: #e0e0e0;
+  vertical-align: sub;
+  color: rgba(0,0,0,.5);
+}
+.vertical-divider {
+  display: inline-block;
```

Comment:
make these styles more specific 
'lf-back-button-vertical-divider'

if button is arrow back, make class 'lf-doc-arrow-back'